src/main/java/com/esri/samples/grouplayers/group_layers/README.md

```diff
@@ -0,0 +1,45 @@
+<h1>Group layers</h1>
+
+<p>Group a collection of layers together and toggle their visibility as a group.</p>
+
+<p><img src="GroupLayers.png"/></p>
```

Comment:
after the update from Lucas on accessibility, I think we should be adding alt tags after all? Depends what route we go down. Either basic with alt="Group Layer" or more detailed like alt="Scene view with 3D buildings and a table of contents for selecting group layers". But that's a discussion point for a later date, stick with basic for now?

Reply:
I think we'll hold off on this until after the release. It might have impacts on how the sample viewer renders the image.